app/workers/articles/update_page_views_worker.rb

```diff
@@ -0,0 +1,25 @@
+module Articles
+  class UpdatePageViewsWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority,
+                    lock: :until_executing,
+                    on_conflict: :replace,
+                    retry: 10
+
+    def perform(create_params)
+      PageView.create(create_params)
+
+      article = Article.find(create_params["article_id"])
+      updated_count = article.page_views.sum(:counts_for_number_of_views)
+      if updated_count > article.page_views_count
+        article.update_column(:page_views_count, updated_count)
+      end
+
+      Articles::UpdateOrganicPageViewsWorker.perform_at(
```

Comment:
For context: the old method in `PageViewsController` called a method called `update_organic_page_views` here. The relationship between the two was 12.5% chance (`rand(8)`) and 1% chance (`rand(100)`), or roughly 12x more likely for page view updates vs organic page view updates. So I kept a similar relationship between the two workers, updates every 5 minutes, organic page view updates hourly.

Reply:
Would be great to transfer this comment into an inline comment in both places, this is context that's precious

Reply:
Good call, added.